examples/twobee/twobee/app.py

```diff
@@ -0,0 +1,38 @@
+import random
+import toga
+from toga.style.pack import Pack, COLUMN, CENTER
+
+HEADS = toga.Image('https://beeware.org/static/images/brutus-270.png')
+TAILS = toga.Image('https://beeware.org/project/projects/libraries/toga/toga.png')
+TOSS = toga.Image('https://beeware.org/project/projects/tools/cricket/cricket.png')
+IMG_SIZE = toga.style.Pack(height=75, width=75)  # TODO be nice if didn't have to specify size
+
+
+def build(_):
+    coins = (toga.ImageView(HEADS, style=IMG_SIZE), toga.ImageView(TAILS, style=IMG_SIZE))
+    announce = toga.Label(text="")
+    buttons = (
+        toga.Button("Two Bee", on_press=lambda b: choose(True)),
+        toga.Button("Two Not Bee", on_press=lambda b: choose(False))
+    )
+    content = toga.Box(children=[announce, toga.Box(children=coins), toga.Box(children=buttons)],
+                       style=Pack(direction=COLUMN, padding=40, alignment=CENTER))
+
```

Comment:
From a formatting point of view, this would be easier to consume as: 
```
content = toga.Box(
    children=[
        announce, 
        toga.Box(children=coins), 
        toga.Box(children=buttons)
    ],
    style=Pack(direction=COLUMN, padding=40, alignment=CENTER)
)
```